ft232/mpsse.py

```diff
@@ -0,0 +1,555 @@
+        """
+        self.__setBitModeMPSSE()
+        self.__PinDirections = PinDirections
+        self.__PinValues = PinValues
+        self.__setPinConfig()
+
+    def __dataCommand(self, ReadWriteData, AsBitsBytes, WriteClkEdge, BitOrder, TmsOnOff):
+        """ Before writing or reading data a Commandbyte has to be sent in advance which defines how the data is sent or received and how it should be interpreted. After sending the Commandbyte the Databytes are expected. 
+        This function implements chapter 3.2 - Data Shifting Commands.
+        The return is one of the OPCODEs in the tables 3.3 - MSB FIRST, 3.4 - LSB First and 3.5 - TMS Commands.
+        """
+        return (TmsOnOff << 6) | (ReadWriteData << 4) | (BitOrder << 3) | (AsBitsBytes << 1) | WriteClkEdge
+
+    def __pinConfigCommand(self, ConfigHighLowPins, SetGetPinConfig):
+        """ To configure the pins or read, wheather they are inputs or outputs and if their initial state is HIGH or LOW a Commandbyte has to be sent in advance which defines if the High Pins (pins ADBUS 7-0) or the Low Pins (pins ACBUS 7-0) are set up or read from. After sending the Commandbyte a Valuebyte is expected, which defines the initial state of the pins (HIGH or LOW) and a Directionbyte, which defines the direction of the pins, weather they are INPUTS or OUTPUTS.
+        This function implements chapter 3.6 - Set / Read Data Bits High / Low Bytes.
+        The return is one of the OPCODEs in the tables 3.6 - Set / Read Data Bits High / Low Bytes.
+        """
+        return 0x80 | (ConfigHighLowPins << 1) | SetGetPinConfig
+
+    def enableLoopback(self):
+        """ When Loopback is enabled TDI/DO and TDO/DI pins are connected with eachother internally to transfer data without an external device. This is for testing.
+        This function implements chapter 3.7 - Loopback Commands.
+        It does not return anything but directly writes the Commandbyte to the device.
+        """
+        self.__Ft232.write(bytes([0x84]))
+
+    def disableLoopback(self):
+        """ This disables Loopback. See enableLoopback() for mor details
+        This function implements chapter 3.7 - Loopback Commands.
+        It does not return anything but directly writes the Commandbyte to the device.
+        """
+        self.__Ft232.write(bytes([0x85]))
+
+    def __setClockDivisor(self, Divisor):
+        """ The Clock of the MPSSE device can be changed by setting a divisor, which lowers the Baseclock of the device by following equation:
+        Clock = Baseclock / (( 1 + Divisor ) * 2)
+        This should not be called directly. Instead use _setClock().
+        This function implements chapter 3.8 - Clock Divisor.
+        It does not return anything but directly writes the Commandbyte and new Divisor to the device.
+        """
+        DivisorLowByte = Divisor & 0xff
+        DivisorHighByte = (Divisor >> 8) & 0xff
+        print("clock divisor set to " + str(Divisor))
```

Comment:
Don't print inside a library, it will just annoy users that want to control their output. Debug messages should be removed.